src/components/Tag/Tag-story.js

```diff
@@ -12,8 +12,8 @@ import Tag, { types as typesList } from '../Tag';
 import TagSkeleton from '../Tag/Tag.Skeleton';
 
 const types = typesList.reduce(
-  (o, type) => ({
-    ...o,
+  (accumulator, type) => ({
```

Comment:
Super super, uber, mega nitpicky, but we tend to abbreviate as `acc`:

```suggestion
  (acc, type) => ({
```

Reply:
No problem @joshblack! I just committed your change 👍